test/jdk/javax/swing/Popup/TaskbarPositionTest.java

```diff
@@ -209,25 +214,38 @@ private void maybeShowPopup(MouseEvent e) {
         }
     }
 
+    // for debugging purpose, saves screen capture when test fails.
+    private static void saveScreenCapture(Rectangle bounds) {
+        BufferedImage image = robot.createScreenCapture(bounds);
+        try {
+            ImageIO.write(image,"png", new File("Screenshot.png"));
```

Comment:
```suggestion
            ImageIO.write(image, "png", new File("Screenshot.png"));
```
Space after the comma.